Corona-Warn-App/src/main/res/values/covid_certificate_attribute_strings.xml

```diff
@@ -13,6 +13,8 @@
 
     <!-- XTXT: Vaccination Certificate Detail Screen Attribute: Name -->
     <string name="covid_certificate_attribute_name" translatable="false">"Name, Vorname / Name, First Name"</string>
+    <!-- XTXT: Vaccination Certificate Detail Screen Attribute: ICAO Name -->
+    <string name="covid_certificate_attribute_icao_name" translatable="false">"Standardisierter Name, Vorname / Standardized Name, First Name"</string>
```

Comment:
Please add the key the German file too ?

Reply:
We don't have a German version of this file: as you can see this file contains both German and English labels simultaneously: `Standardisierter Name, Vorname / Standardized Name, First Name`

Description in this file:
```
<!-- ####################################
        Special string files for the strings
        that should always keep the german / english
        version independent from the selected system language
    ###################################### -->
```

Reply:
Ah you are  right!